README.md

```diff
@@ -62,6 +62,12 @@ If you want to collect a set of results that can then later be used to create gr
 CUSTOM_BENCH_REGEX="_stable.*random-" python util/run_benchmarks.py my_test_zen3
 ```
 
+To create graphs from this data, you can use the `graph_all.py` script: \
+Note: An older version of [bokeh](https://pypi.org/project/bokeh/2.4.3/) must be installed, e.g. version 2.4.3
```

Comment:
I'd rather mention the requirements file:

To run the `graph_all.py` script you need to first install the dependencies as specified in `requirements.txt` e.g. on Linux:

```
python -m venv venv
source ./venv/bin/active
pip install --requirement util/graph_bench_result/requirements.txt
```